Felienne/hedy
# Makes sure empty code isn't parsed at all

**Description**
We add a validation to the front-end to make sure there is actual code before parsing and returning some success message. This reduces server-load, achievement spamming and makes the UI less 'flashy' when running an empty program.

**Fixes**
This PR fixes #3290.

**How to test**
Run an empty program. Verify that no POST is made to the server and no success message is returned either.
